src/main/java/com/group6/harmoniq/models/Quiz.java

```diff
@@ -25,6 +25,10 @@ public Quiz(String question_path, String answer, String option1, String option2,
     }
     public Long getId() {
         return id;
+    }
+     //For testing purposes
+     public void setId(Long id) {
```

Comment:
Adding a new methond or exposing a readonly field just for testing is usually not a good approach. The tests should be adapted to the model or unit of code that we are trying to test, not other way around. In this case, we can skip the getId test if its not possible without adding this method to the model